src/integrationTest/java/uk/gov/hmcts/darts/testutils/stubs/DartsDatabaseStub.java

```diff
@@ -205,6 +207,7 @@ public void clearDatabaseInThisOrder() {
         // securityGroupRepository.deleteAll(securityGroupBin);
         // securityGroupBin.clear();
         courthouseRepository.deleteAll();
+        regionRepository.deleteAll();
```

Comment:
I don't think we should delete standing data: region
What do you think here @tonymort ?

Reply:
Yeah, that is the general pattern, theres example of where we do this however which is wrong and should be fixed. Going forward however though I agree lets not introduce more this.  Means we should create/delete new regions in integration tests and only rely on the standing data.

Reply:
I could see the courthouses were deleted and took it as similar standing data. I can remove and push again.

Reply:
Removed, but had knock on issues. So I added again. We are probably better doing this consistently and removing the clearing of all standing data together on a seperate ticket.

Reply:
I'm assuming @tonymort means to look at your integration test - and you would basically expect to see the 7 regions returned in the GET request (since we wouldn't be deleting them).
They are standing data in the sense that we have purposefully loaded them via our script and wouldn't expect them to be added by a service (endpoint).